src/Microsoft.Health.SqlServer/Features/Schema/SchemaUpgradeRunner.cs

```diff
@@ -48,7 +48,8 @@ public async Task ApplySchemaAsync(int version, bool applyFullSchemaSnapshot, Ca
         {
             _logger.LogInformation("Applying schema {version}", version);
 
-            if (!applyFullSchemaSnapshot)
+            // Eliminate the need to insert schema version from 6.sql, 7.sql and so on.
```

Comment:
> // Eliminate the need to insert schema version from 6.sql, 7.sql and so on. [](start = 11, length = 76)

Any chance you can rephrase comment here from instead of what it is doing to why it is doing it?

Reply:
Thanks, fixed.